.github/workflows/ubuntu-cminpack-install.yaml

```diff
@@ -41,11 +36,6 @@ jobs:
       
       - name: Test cminpack
         run: ctest --test-dir ${{ env.CMINPACK_BUILD_DIR }} --build-config ${{ matrix.CMAKE_BUILD_TYPE }}
-        if: ${{ matrix.USE_BLAS == 'OFF' }}
```

Comment:
It's OK to exclude one test for OpenBLAS, until someone contributes a better testing mechanism. Maybe Fortran BLAS passes the tests, but it's not used anymore